Formula/signal-cli.rb

```diff
@@ -0,0 +1,20 @@
+class SignalCli < Formula
+  desc "CLI and dbus interface for WhisperSystems/libsignal-service-java"
+  homepage "https://github.com/AsamK/signal-cli"
+  url "https://github.com/AsamK/signal-cli/releases/download/v0.6.2/signal-cli-0.6.2.tar.gz"
+  sha256 "d917507211b3419e271674df7f03c8189921e2e78a389479a612847ddc8f1614"
+
+  bottle :unneeded
+  depends_on :java => "1.7+"
+
+  def install
+    libexec.install Dir["lib", "bin"]
+    (bin/"signal-cli").write_env_script libexec/"bin/signal-cli", Language::Java.java_home_env("1.7+")
+  end
+
+  test do
+    # checks class loading is working and version is correct
```

Comment:
Any chance we can have a test that does a little more? Like try to connect / perform some operation, and check that it works (or fails)?

Reply:
The `signal-cli link` command was the only one I found that exercised the cryptographic aspects and didn't require credentials configured. It was working on Homebrew CI, but something changed recently and it's now failing (network operations seem prohibited), so I removed it in 7a3900e. All of the other commands require network or credentials.

Reply:
Network operations are permitted in CI.

Reply:
OK, I'll revert that test change and we can debug networking. Thanks. :)

Reply:
8 seconds (vs 2) seems to have worked.